globals.d.ts

```diff
@@ -0,0 +1,70 @@
+declare module "tiny-markdown-editor" {
+  type CursorPosition = { col: number; row: number; node: string };
+
+  type CommandName =
+    | "bold"
+    | "italic"
+    | "strikethrough"
+    | "code"
+    | "h1"
+    | "h2"
+    | "ul"
+    | "ol"
+    | "blockquote"
+    | "hr"
+    | "insertLink"
+    | "insertImage";
+
+  type ChangeEvent = { content: string; linesDirty: boolean[] };
+
+  type SelectionEvent = {
+    focus: CursorPosition;
+    anchor: boolean;
+    commandState: Record<string | CommandName, boolean>;
+  };
+
+  type Listener<T extends "change" | "selection"> = (
+    event: T extends "change" ? ChangeEvent : SelectionEvent
+  ) => void;
+
+  type EditorParams = { element: string; content?: string; textarea?: string };
```

Comment:
`element` is a DOM element

Reply:
So is `textarea`

Reply:
My bad! I glossed over this one despite having the correct code on my local.